unit/testing-utils/require_symbol.h

```diff
@@ -0,0 +1,25 @@
+/*******************************************************************\
+
+ Module: Unit test utilities
+
+ Author: DiffBlue Limited. All rights reserved.
+
+\*******************************************************************/
+
+/// \file
+/// Helper functions for requiring properties of symbols
+
+#include <util/symbol.h>
+#include <util/std_types.h>
+
+#ifndef CPROVER_TESTING_UTILS_REQUIRE_SYMBOL_H
+#define CPROVER_TESTING_UTILS_REQUIRE_SYMBOL_H
+
+// NOLINTNEXTLINE(readability/namespace)
+namespace require_symbol
+{
+  const class_typet &require_complete_class(
+    const symbolt &class_symbol);
+}
+
+#endif
```

Comment:
Nit - it's usually nice with `#endif`s to include the DEFINE that they are ending as a comment at the end of the line, e.g. ```
#endif // CPROVER_TESTING_UTILS_REQUIRE_SYMBOL_H
```
You'll see this pattern in other header files. While it's not a big deal with a header file as small as this, it's a good habit to get into because if you ever end up working on a file with multiple nested `#ifdef`s, it really helps to disambiguate the endings :-)